docs/getting-started/user-guides/testnet-voting.md

```diff
@@ -0,0 +1,183 @@
+
+From the Start Menu, open `Decred Testnet`. This will launch `Paymetheus`, and a new Command Prompt window will open and run `dcrd.exe`. If this is the first time running the testnet daemon, it will take awhile to sync to the testnet blockchain. 
+
+In the `Paymetheus` window, you'll be greated by a "Connect to dcrd" dialog. Keep the defaults and press the continue button. The next view will have two buttons, "Create a new wallet" and "Restore wallet from seed." For this guide, it will be assumed you do not already have a seed you wish to restore.
+
+After clicking "Create a new wallet," you'll be greeted with information regarding your new wallet seed. Record your seed, put it in a safe place, and never share it with anyone. You will also need to re-enter it once you press the CONTINUE button.
+
+After you've typed in your seed, the Encrypt Wallet view will be next. Enter a private passphrase as the directions explain. Press ENCRYPT. Paymetheus will then begin to create your wallet. Once it is created, it will open to your wallet overview page.
+
+> Step 3: Register at Stakepool Website
+
+While waiting for your node/wallet to sync, visit [https://teststakepool.decred.org](https://teststakepool.decred.org) and register for a new account. 
+
+> Step 4: Acquire Testnet Coins
+
+Next, you'll need to acquire Testnet coins in order to buy Testnet tickets. There is am official Testnet faucet located at [https://faucet.decred.org](https://faucet.decred.org). To get a new address from Paymetheus, click the "Request payment" tab on the navigation menu. Clicking the "GENERATE ADDRESS" button will result in an address that should begin with "Ts". Copy and paste that address into the faucet and you should recieve your coins.
+
+> Step 5: Purchase Testnet Tickets
+
+Click the "Purchase tickets" tab within the Paymetheus navigation menu. You will see 7 form fields within the page. All of the defaults can be used for purchasing tickets **except** the "Stake pool preference". Click the "Manage pools" button. You need to enter the API key for your account at the testnet stakepool. To do this, simply visit [https://teststakepool.decred.org/settings](https://teststakepool.decred.org/settings) - your API Token should be the first item on the page. Enter it into the API Key field within Paymetheus and press Save. Your 1-of-2 multi-sig script will be automatically generated and you can press Close. From here, select teststakepool.decred.org from the Stake pool preference drop down and press the Purchase button begin purchasing tickets! Note: the ticket difficulty is equal to the cost per ticket, so make sure you have enough testnet coins to purchase at least one.
+
+> Step 6: Set The Votebits of Your Tickets via Stakepool 
+
+When you buy tickets via a stakepool, you automatically give that pool control of the votebits. The pool can then vote whichever way it pleases. (Note, if you purchase tickets without a stakepool, your ticket votebits default to 1)
```

Comment:
tickets aren't bought via a stakepool, they are bought with voting rights delegated to the stakepool.  because of the delegated voting rights, the pool can vote any way it pleases